<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingCell</class>
 <widget class="QWidget" name="SettingCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>IBeamCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="setting_header">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/new/headers/images/headers/11.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="setting_username">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0); border: 0px 1px, 0px, 0px; border-bottom: 1px solid white; </string>
   </property>
  </widget>
  <widget class="QLabel" name="setting_ip">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setting_password">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0); border: 0px 1px, 0px, 0px; border-bottom: 1px solid white; </string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setting_save">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0); border: 1px solid white; border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="setting_id_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="setting_id_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <zorder>setting_ip</zorder>
  <zorder>setting_header</zorder>
  <zorder>setting_username</zorder>
  <zorder>setting_password</zorder>
  <zorder>label_3</zorder>
  <zorder>setting_save</zorder>
  <zorder>setting_id_2</zorder>
  <zorder>setting_id_3</zorder>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
